README.md

```diff
@@ -12,17 +12,23 @@ Please see our [contributing guidelines](.github/CONTRIBUTING.md)
 
 ### [Installing](https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages#installing-a-package)
```

Comment:
no need for the link if we don't wanna use Github Package Registry.

Reply:
we'll get there